app/views/wiki/_wikis.html.erb

```diff
@@ -1,35 +1,39 @@
-<% wikis ||= @wikis #accept local if present, default to instance %>
-<% if local_assigns[:digest] and wikis.size == 0 %>
-  <div class="alert alert-warning" role="alert">
-    You haven't subscribed to any topic yet. Visit <a href="https://publiclab.org/subscriptions">https://publiclab.org/subscriptions</a> to subscribe to more topics.
-  </div>
-<% else %>
-
-  <table class="table">
-    <tr>
-      <th><a href = "?sort=title"> <%= translation('wiki._wikis.title') %></a> <i class="fa fa-sort" aria-hidden="true"></i></th>
-      <th>URL</th>
-      <th><a href = "?sort=last_edited"> <%= translation('wiki._wikis.last_edited') %></a> <i class="fa fa-sort" aria-hidden="true"></i></th>
-      <th><a href = "?sort=edits"> <%= translation('wiki._wikis.edits') %></a> <i class="fa fa-sort" aria-hidden="true"></i></th>
-      <th><a href = "?sort=page_views"> <%= translation('wiki._wikis.page_views') %></a> <i class="fa fa-sort" aria-hidden="true"></i></th>
-      <th><a href = "?sort=likes"> <%= translation('wiki._wikis.likes') %></a> <i class="fa fa-sort" aria-hidden="true"></i></th>
```

Comment:
@mittal-parth The test is trying to assert that the element on line 15 exists but since we've removed it, the assertion fails. What you need to do is pick another element that you can use to test that this partial is displayed when the user visits `/wiki`.

Reply:
Ah got it, thanks! Updated the test.